src/index.js

```diff
@@ -0,0 +1,102 @@
+const registerEvents = () => {
+  console.log('loaded');
+
+  let currentTemp = 71;
+
+  const resetButton = document.getElementById('cityNameResetButton');
+  resetButton.addEventListener('click', () => {
+    document.getElementById('cityNameInput').value = 'Seattle';
+    updateCityName();
+  });
+
+  const updateCityName = () => {
+    console.log('Updating City name');
+    const inputCity = document.getElementById('cityNameInput');
+    const headerCityNameSpan = document.getElementById('headerCityName');
+    headerCityNameSpan.textContent = inputCity.value;
+  };
+
+  const inputCity = document.getElementById('cityNameInput');
+  inputCity.addEventListener('input', (event) => {
+    console.log('Got an input event');
+    updateCityName();
+  });
+
+  const skySelector = document.getElementById('skySelector');
+  skySelector.addEventListener('change', (event) => {
+    console.log('sky selector changed');
+    console.log(event);
+    console.log(skySelector.value);
+    updateWeatherGarden();
+  });
+
+  const tempUp = document.getElementById('tempUp');
+  tempUp.addEventListener('click', () => {
+    console.log('Temp up clicked');
+    currentTemp++;
+    updateTemperatureValue();
+  });
+
+  const tempDown = document.getElementById('tempDown');
+  tempDown.addEventListener('click', () => {
+    console.log('Temp down clicked');
+    currentTemp--;
+    updateTemperatureValue();
+  });
+
+  const updateTemperatureValue = () => {
+    const tempValue = document.getElementById('tempValue');
+    tempValue.textContent = currentTemp;
+    tempValue.classList = [];
+    tempValue.classList.add(getColorClassForTemp(currentTemp));
```

Comment:
To clear out the classes and set a single class, we can use the `className` attribute instead.

```js
    tempValue.className = getColorClassForTemp(currentTemp);
```

Great use of a helper to calculate the class name itself!